try {
    var a = [3, 4, 12, 6, 33];
    console.log(a);
    // it is undefined. so invoke catch error.
    console.log(b);
} catch (e) {
    console.log(e);
}